hphp/runtime/vm/jit/abi-arm.h

```diff
@@ -144,6 +165,46 @@ inline jit::ConditionCode convertCC(vixl::Condition cc) {
   return mapping[cc];
 }
 
+enum class StatusFlags : uint8_t {
+  None = 0,
+  N    = 1 << 3,
+  C    = 1 << 2,
+  Z    = 1 << 1,
+  V    = 1,
+  CZ   = C | Z,
+  NV = N | V,
+  NZV = N | Z | V,
+  NotC = N | Z | V,
+  NotV = N | C | Z,
+  All  = N | C | Z | V,
+};
```

Comment:
These are the ARM-specific status flag combinations tested with the 16 different ARM condition codes.